sys/reaper/reaper_unix.go

```diff
@@ -58,9 +58,10 @@ func (s *subscriber) do(fn func()) {
 
 // Reap should be called when the process receives an SIGCHLD.  Reap will reap
 // all exited processes and close their wait channels
-func Reap() error {
+func Reap(bundlePath string) error {
 	now := time.Now()
-	exits, err := reap(false)
+	exits, _ := reap(false)
```

Comment:
for non darwin.. it looks like this just forces Reap to return nil no matter if an error occurred in reap()

Reply:
Thanks, you're right.  It should try to preserve previous behavior.
I'll fix this in next patch.